src/resources/vue/components/terminal-scan.vue

```diff
@@ -1,35 +1,58 @@
 <template>
 	<div>
-		<Box>
+		<Box :direction="direction">
 			<div class="content">
-				<svg class="scan-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
-					<path d="M15 3H21V8H19V5H15V3ZM9 3V5H5V8H3V3H9ZM15 21V19H19V16H21V21H15ZM9 21H3V16H5V19H9V21ZM3 11H21V13H3V11Z"></path>
-				</svg>
-				<span class="text">Scan een artikel om te lenen</span>
+				<Icon v-if="text" icon="scan-icon" size="52px" />
+				<span v-if="text" class="text">{{ text }}</span>
+			</div>
+			<div class="form">
+				<div class="form-input">
+					<Icon v-if="!text" :icon="icon" size="52px" /> <!-- v-if="direction == row" -->
+					<Input type="text" :placeholder="placeholder" class="text-medium" />
+				</div>
+				<Button content="Zoek" :buttonAction="searchInput" />
 			</div>
-			<Input type="text" placeholder="Voer wat in" class="text-medium" />
-			<Button content="Zoek" :buttonAction="searchInput" />
 		</Box>
 	</div>
 </template>
 <script>
 	import Box from '../components/Box.vue';
 	import Button from '../components/Button.vue';
 	import Input from '../components/Input.vue';
+	import Icon from '../components/Icon.vue';
 
 	export default {
 		name: 'index',
 		components: {
 			Box,
 			Button,
-			Input
+			Input,
+			Icon
 		},
 		methods: {
 			searchInput() {
 				// TODO: Implement search input
```

Comment:
als dit moet blijven resolve zonder aanpassing. zo niet weg.

Reply:
removed console log, but kept the todo